### PR TITLE
All texts translated to Chinese

### PR DESCRIPTION
**Pull request by Cyrillya**
Translated all texts into Chinese
Some new translate keys were set. They are in the en-US.hjson and zh-Hans.hjson

Below is the Chinese version of the mod description. It will be good if you set the Chinese description on the Steam Workshop. (It is able to set different descriptions for different languages on Workshop)

--- Mod Display Name ---
`Magic Storage 魔法储存`
(I suggest posting the English name as well)

--- Mod Description ---
```
中文翻译由 [url=https://steamcommunity.com/id/cyrilliy/]-Cyril-[/url] 提供

你是否对箱子里一堆杂乱玩意感到烦恼？你是否放物品从来乱放，每次找东西都要翻遍你所有的箱子？这个模组完全可以解决你的这些问题！

本模组会一劳永逸地解决你的储存问题，甚至可以构建一个物品储存网络来存储你的一切物品。你可以从存储核心来访问你的存储网络，并且可以通过存储连接器来设置不同的存储访问点，以在不同的地方随时取放你的物品。存储核心同时兼备搜素和过滤功能，你可以搜索你的物品，也可以用不同的过滤器找到某一特定类型的物品。此外，通过制作存储单元，你甚至可以直接在存储网络内制作物品！

随着你的游戏进度，你可以用不同的升级珠来升级你的存储容量。你在大前期就可以建立存储网络，但能力有限，随着更多的Boss被击败，取得了更多的后期材料，你的存储网络就可以升级，以取得更大容量与更多功能。

同时，本模组也具备很多合并制作站，你可以将不同的制作站合成，制作一个兼具多种制作站功能的组合制作站，再也不用为制作站太多而烦恼！

=== 重要提示 ===
报告Bug请通过此链接加入Discord服务器： https://discord.gg/FemPG7eev4
（如果你在中国大陆区域，你也许需要通过修改hosts或挂VPN才能访问）

1.4魔法储存模组合并了1.3版本魔法储存Magic Storage)与魔法储存Extra(Magic Storage - Extra)的功能

=== 魔法储存制作团队 ===
blushiemagic
- 原作者
absoluteAquarian
- 现作者，模组维护者
wvlad
- 负责重要的修复工作，并将其纳入Magic Storage - Extra
Exterminator
- Magic Storage - Extra 的作者
AdipemDragon
- 主画师
Yowave
- 在2021年10月20日至2021年12月15日期间，为本模组修复Bug并优化
```

### CODE REVIEW

**Comment by ExterminatorX99** [comment]
Please undo the formatting changes, brace style and tab/space indentation

**Comment by Crapsky233** [comment]
> Please undo the formatting changes, brace style and tab/space indentation

Fixed now

File: Stations/CombinedStations1Item.cs [hunk 1]
```diff
@@ -31,7 +43,7 @@ public override void GetItemDimensions(out int width, out int height)
 			height = 30;
 		}
 
-		public override void AddRecipes()
+		public override void AddRecipes()
```
[the remark this answers is not in the document]
Comment (Crapsky233) [reply Stations/CombinedStations1Item.cs]: Forgot to delete that...

**Comment by Crapsky233** [comment]
I ported all translations (excluding item names and tooltips) to hjsons in Localization/
This way the translation work can be done better

**Comment by Cyrillya** [comment]
> I just merged a pretty large PR, so please fix the merge conflicts.

Fixed

**Pull request merged**